ci/test_shell.sh

```diff
@@ -11,7 +11,7 @@ if ! [ -x "$(command -v bats)" ]; then
 fi
 if ! [ -x "$(command -v shellcheck)" ]; then
     echo "=== install shellcheck ==="
-    wget https://storage.googleapis.com/shellcheck/shellcheck-stable.linux.x86_64.tar.xz
```

Comment:
could you revert to this?
I don't know why but the problem of the spellcheck seems to come from this.
Another possibility is to fix all shell errors.
I just saw them, and we could easily fix them.